packages/playground/src/pages/Dropin/Dropin.js

```diff
@@ -99,6 +99,12 @@ const initCheckout = async () => {
                     }
                 }
             },
+            applepay: {
+                configuration: {
+                    merchantName: 'Adyen Test merchant',
+                    merchantIdentifier: '000000000200001'
+                }
```

Comment:
Aren't we getting this in the `/paymentMethods` response?

Reply:
Not always, this is useful for testing (as we have on the Wallets page).